test/no-deploy/timelock.ts

```diff
@@ -163,6 +158,59 @@ describe("Timelock", () => {
     });
   });
 
+  describe.only("Cancelling transaction", async () => {
```

Comment:
@MichalSzreder beware of leaving these ".only"s in the code

Reply:
you should also test if the right event was emitted. The example how to get the events you van find in `extractDataHashAndTxHash`

Reply:
Removed .only and tested events where possible